extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegesBuilder.scala

```diff
@@ -273,6 +308,18 @@ object PrivilegesBuilder {
           inputObjs += databasePrivileges(db.get)
         }
 
+      case "CreateNamespace" =>
```

Comment:
the command here is out of the scope of this PR, at least it does match the PR title

Reply:
Change the PR and issue with extended command list for covering both databases/namespaces and tables.